ontologies/vocabs.ttl

```diff
@@ -588,8 +574,10 @@ Specifies the ChaCha20-Poly1305 stream cipher, as defined in [RFC7539]."""^^xsd:
 
 gist:_EncryptionAlgorithm_mime-type-indicated
 	a gist:EncryptionAlgorithm ;
-	rdfs:label "Mime Type Indicated"^^xsd:string ;
-	skos:altLabel "Mime-Type-Indicated"^^xsd:string ;
+	skos:altLabel
+		"Mime Type Indicated"^^xsd:string ,
+		"Mime-Type-Indicated"^^xsd:string
```

Comment:
Is this helpful?
Also, should there not be a stixTerm? I guess that is a different git issue?